streaming/src/main/scala/io/gearpump/streaming/appmaster/TaskManager.scala

```diff
@@ -260,6 +248,8 @@ private[appmaster] class TaskManager(
       }
   }
 
+  when(Recovery)(startTasks orElse ignoreClockEvent)
```

Comment:
When recovery, I think it's using the recovery state, whose TaskRegistry is empty.